docs/user/adding-new-metrics.md

```diff
@@ -6,54 +6,61 @@ The format of that file is documented [here](https://mozilla.github.io/glean_par
-    expires: 2019-06-01  # <-- Update to a date in the future
+If the value is a string, use a [string metric](metrics/string.html).  If you want to store multiple strings in a group, use a [string list metric](metrics/string_list.html).
 
-  double_click:
-    ...
+If you need to record a closely-related group of strings, or you don't know the things the strings are related to at build time, use a [labeled string metric](metrics/labled_strings.html).
 
-category2.subcategory:  # Categories can contain subcategories
-  metric:
-    ...
+For all of these simple value types, you should consider how the value changes in the application and how that relates to the ping it is being sent in. When the ping is submitted, it will contain the "last known" value for the metric, and then the value is reset according to its [lifetime](#What-is-the-lifetime-of-the-data).
 
-```
+If you need to know the current state of a value, such as a user preference, to provide context for other metrics, be aware that the lifetime of the value may not be the same as the lifetime of the ping it is in. You can address this either with a custom ping that matches the lifetime of the value, or by using a user lifetime metric so the value will persist across pings.
 
-The details of the metric parameters are described in [metric parameters](metric-parameters.md).
+### Are you counting things?
 
-The `metrics.yaml` file is used to generate code in the target language (e.g. Kotlin, Swift, ...) that becomes the public API to access your application's metrics.
+If you want to know how many times something happened, use a [counter metric](metrics/counter.html).  If you are counting a group of related things, or you don't know what all of the things to count are at build time, use a [labeled counter metric](metrics/labeled_counter.html).
+
+If you need to know when the things being counted happened relative to other things, use an [event](metrics/event.html).
+
+### Are you measuring time?
+
+If you need to record an absolute time, use a [datetime metric](metrics/datetime.html). Datetimes are recorded in the user's local time, along with a timezone offset from UTC.
+
+If you need to record how long something takes you have a couple of options:
+
+If you need to measure the total time spent doing a particular task, look to the [timespan metric](metric/timespan.html). Note that it can only be used on a single thread.
 
-## Recommendations for defining new metrics
+If you need to measure the relative occurrences of many timings, use a [timing distribution](metric/timing_distribution.html). It builds a histogram of timing measurements, and is safe to use from multiple threads.
 
-“There are only two hard things in Computer Science: cache invalidation and naming things” -- attributed to Phil Karlton.
+If you need to know the time between distinct events that aren't a simple "begin" and "end" pair, consider using an [event](metric/event.html).
```

Comment:
At first I was... "eugh" with this sentence. But it's really a legitimate use-case I didn't think about! Good catch.

Reply:
Any "Eugh" is a candidate for a new metric type to me. Let's say a "timing list" where each step in the multi-part process is represented as a count of nanos from the first `start()`. ...but if what we're hoping is to hear from people who want this metric type, we can't provide a workaround for them in this guide.

Reply:
I think if you want to answer, how long was spent between performing action A and action B, events are exactly the thing that was designed for.  I think the proposed timing list metric might be redundant to all of that.  Let me reword this to be a little more on-the-nose about the events use case.